component/tests/vshn.yml

```diff
@@ -152,7 +152,7 @@ parameters:
                   appuio.io/node-class: "plus"
               note: "Will be scheduled on APPUiO Cloud plus nodes"
         minio:
-          enabled: false
+          enabled: true
```

Comment:
Can you reset this? Usually we use kindev's minio as it's wired into the ingress.

Reply:
Done